codegen/luau/src/analyzer/into_string.rs

```diff
@@ -71,13 +71,13 @@ impl IntoNameTuple for UnOpType {
 			Self::Nearest_F64 => ("rt_nearest", "f64"),
 			Self::Sqrt_F64 => ("math", "sqrt"),
 			Self::Wrap_I32_I64 => ("rt_wrap", "i32_i64"),
-			Self::Truncate_I32_F32 => ("rt_truncate", "i32_f32"),
-			Self::Truncate_I32_F64 => ("rt_truncate", "u32_f64"),
-			Self::Truncate_U32_F32 => ("rt_truncate", "u32_f32"),
-			Self::Truncate_U32_F64 => ("rt_truncate", "u32_f64"),
-			Self::Truncate_I64_F32 => ("rt_truncate", "i64_f32"),
+			Self::Truncate_I32_F32 => ("rt_truncate", "i32_f64"),
+			Self::Truncate_I32_F64 => ("rt_truncate", "i32_f64"),
+			Self::Truncate_U32_F32 => ("rt_truncate", "f64"),
+			Self::Truncate_U32_F64 => ("rt_truncate", "f64"),
```

Comment:
These also really should be `i32_f64`, otherwise there's no truncation happening here.
`rt_truncate_i32_f64` already treats numbers as unsigned by using `bit.bor`, so you should use it here.

Reply:
alright ive changed it